R/plotCorrelations.R

```diff
@@ -0,0 +1,319 @@
+#' \code{preDraw} Helper function.
+#' Helper function for the co-expression correlation and semantic similarity correllation heat maps
+#' @param fileName (character)     if not NULL - it will use the Cairo package to create a
+#'                                 device. It checks the file extension to
+#'                                 instantiate appropriate device;
+#'                                 otherwise - it does nothing (graphs are plotted on screen)
+#' @return (NULL)
+#' @author \href{https://orcid.org/0000-0002-8778-6442}{Denitsa Vasileva} (aut)
+#' [Cairo package](https://CRAN.R-project.org/package=Cairo
+preDraw <- function(fileName) {
+  if (is.null(fileName)) {
+    return(invisible(NULL))
+  }
+  # is the file extension pdf?
+  if (grepl(".pdf$", fileName)) {
+    Cairo::CairoPDF(
+      file = fileName,
+      width = 1920,
+      height = 1080,
+      res = 92,
+      pointsize = 12
+    )
+    return(invisible(NULL))
+  }
+  # otherwise - default to png
+  if (!grepl(".png$", fileName)) {
+    fileName <- paste0(fileName, ".png")
+  }
+  # Cairo Package https://CRAN.R-project.org/package=Cairo
+  Cairo::CairoPNG(
+    file = fileName,
+    width = 1920,
+    height = 1080,
+    res = 92,
+    pointsize = 12
+  )
+  return(invisible(NULL))
+}
+
+#' Helper function for the co-expression correlation & semantic similarity correlation heat maps
+#' @param fileName - if not NULL - it will call dev.off to save the file
+#' otherwise - it prompts for user input - so the user can see the graph
+#' before the next one is drawn
+#' @return (NULL)
+#' @author \href{https://orcid.org/0000-0002-8778-6442}{Denitsa Vasileva} (aut)
+postDraw <- function(fileName) {
+  if (is.null(fileName)) {
+    readline(prompt = "Press enter to continue...")
+  } else {
+    dev.off()
+  }
+  return(invisible(NULL))
+}
+
+
+#'
+#' \code{installHumanGenomeAnnotation()} installs Human Genome Annotation
+#' as per Dr. Stipe's email and
+#' https://bioconductor.org/packages/release/data/annotation/html/org.Hs.eg.db.html
+#'
+installHumanGenomeAnnotation <- function() {
```

Comment:
You don't need to check for BiocManager - it should be in `Imports:` - or at least `Suggests:`.